indico/modules/logs/models/entries.py

```diff
@@ -153,6 +153,10 @@ def render(self):
     def __repr__(self):
         return format_repr(self, 'id', type(self).link_fk_name, 'logged_dt', 'realm', 'module', _text=self.summary)
 
+    @property
+    def locator(self):
+        return {'log_entry_id': self.id}
```

Comment:
Include the `event_id` in here

```suggestion
        return {'event_id', self.event_id, 'log_entry_id': self.id}
```

Reply:
ah, and of course move it to `EventLogEntry` since this is not compatible with other log entry types